lnpbp-0030.md

```diff
@@ -0,0 +1,113 @@
+```
+LNPBP: 0030
+Aliases: RGB30
+Vertical: Smart contracts
+Title: Interface for fungible RGB assets with decentralized issue (RGB-30)
+Authors: Dr Maxim Orlovsky <orlovsky@lnp-bp.org>
+Comments-URI: <https://github.com/LNP-BP/LNPBPs/discussions/140>
+Status: Proposal
+Type: Standards Track
+Created: 2021-06-23
+Updated: 2023-04-20
+Finalized: ~
+Copyright: (0) public domain
+License: CC0-1.0
+```
+
+- [Abstract](#abstract)
+- [Background](#background)
+- [Motivation](#motivation)
+- [Design](#design)
+- [Specification](#specification)
+- [Compatibility](#compatibility)
+- [Rationale](#rationale)
+- [Reference implementation](#reference-implementation)
+- [Acknowledgements](#acknowledgements)
+- [References](#references)
+- [Copyright](#copyright)
+
+
+## Abstract
+
+
+## Background
+
+
+## Motivation
+
+
+## Design
```

Comment:
It would be nice to get a better sense for the high-level plan for decentralized issuance and workflows surrounding it.